string home_folder
string current_folder
string[] current_folders

bool paused

int32 img_count
bool img_random
bool img_overlay
float32 img_delay
bool img_pub_running

int32 vid_count
bool vid_random
bool vid_overlay
bool vid_pub_running


int32 pc_count
int32 pc_max_pub
int32 pc_pub_tfs
bool pc_pub_running



